packages/consumption/src/modules/attributes/local/LocalAttribute.ts

```diff
@@ -177,6 +196,10 @@ export class LocalAttribute extends CoreSynchronizable implements ILocalAttribut
         return !!this.shareInfo;
     }
 
+    public isNotDefault(): this is LocalAttribute & { default: undefined } {
```

Comment:
Be positive :-) In general, give boolean things affirmative names such as `isDefault` or `isSet` as opposed to `isNotDefault` or `isNotSet`. That's not a hard rule by any means but more of an unwritten convention in programming.

https://softwareengineering.stackexchange.com/questions/196830